eo-runtime/src/test/groovy/check-folders-numbering.groovy

```diff
@@ -42,6 +42,9 @@ List<File> allowed = [
   .map { target.resolve(it).toFile() }
   .collect(Collectors.toList())
 
-assert allowed.containsAll(directories)
-
+try {
+  assert allowed.containsAll(directories)
+} catch (AssertionError e) {
+  fail(e);
+}
```

Comment:
@volodya-lombrozo can't you just say:

```
if (!allowed.containsAll(directories)) {
  fail("smth");
}
```

Reply:
@yegor256 I can, but in that case we will loose quite informative error message:
instead of 
```
[ERROR] Failed to execute goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute (project-validate) on project eo-runtime: Execution project-validate of goal org.codehaus.gmaven:groovy-maven-plugin:2.1.1:execute failed: org.apache.maven.plugin.MojoExecutionException: assert allowed.containsAll(directories)
[ERROR]        |       |           |
[ERROR]        |       false       [/home/tardis3/eo/eo-runtime/target/eo/6-transpile, /home/tardis3/eo/eo-runtime/target/eo/new-dir, /home/tardis3/eo/eo-runtime/target/eo/4-resolve, /home/tardis3/eo/eo-runtime/target/eo/1-parse, /home/tardis3/eo/eo-runtime/target/eo/2-optimize, /home/tardis3/eo/eo-runtime/target/eo/5-pre, /home/tardis3/eo/eo-runtime/target/eo/3-pull]
[ERROR]        [/home/tardis3/eo/eo-runtime/target/eo/1-parse, /home/tardis3/eo/eo-runtime/target/eo/2-optimize, /home/tardis3/eo/eo-runtime/target/eo/3-pull, /home/tardis3/eo/eo-runtime/target/eo/4-resolve, /home/tardis3/eo/eo-runtime/target/eo/5-pre, /home/tardis3/eo/eo-runtime/target/eo/6-transpile]
[ERROR] -> [Help 1]
[ERROR] 
```

We will see:
```
RuntimeException: smth
```


I don't think it is worth it.

Reply:
@volodya-lombrozo maybe you can print a better message instead of `smth`. Print something like `"The directory xxx is absent, but expected"` or the other way around: `"The directory xxx is not expected to be here"`